client/src/IndexStyle/normalize.module.scss

```diff
@@ -0,0 +1,180 @@
+html {
+  width: 100vw;
+  line-height: 1.15;
+  -webkit-text-size-adjust: 100%;
+}
+
+body {
+  margin: 0;
+}
+
+main {
+  display: block;
+}
+
+h1 {
```

Comment:
why? This will cause trouble if some one doesn't know about this overrite. create a custom class for this.

Reply:
Okay. Understood.I removed this normalize file and added custom CSS properties.